editor/src/components/canvas/canvas-wrapper-component.tsx

```diff
@@ -156,6 +473,27 @@ export const CanvasWrapperComponent = React.memo(() => {
   )
 })
 
+const AreaSelect = React.memo(({ rectangle }: { rectangle: CanvasRectangle | null }) => {
+  const colorTheme = useColorTheme()
+  if (rectangle == null) {
+    return null
+  }
+
+  return (
+    <div
+      style={{
+        border: `1px solid ${colorTheme.primary.value}`,
+        background: colorTheme.primary10.value,
+        position: 'absolute',
+        width: rectangle.width,
+        height: rectangle.height,
+        left: rectangle.x,
+        top: rectangle.y,
+      }}
+    />
+  )
+})
```

Comment:
if you set `.displayName` on this, it'll show up with that name further down in re-rendering screenshot

Reply:
https://github.com/concrete-utopia/utopia/pull/3750/commits/ae5f541ea826bd44ded86ec29c14312c4519656c